src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -228,7 +228,7 @@ public UUID createAndLock(Dataset dataset, String flightId) throws IOException,
         return datasetId;
     }
 
-    protected String getExclusiveLock(UUID id) {
+    public String getExclusiveLock(UUID id) {
```

Comment:
This method should stay protected. It makes it so that only tests in the same package can call it, not other API code outside of the package.